### DISCUSSION TITLE
Remove disabled accounts from Assignee list Cases

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#8161] [Unsupported Versions] [answered]
Is it possible to remove disabled SOC accounts from the assignee list in cases, I know you can't/shouldnt delete SOC accounts which I'm fine with but the accounts still show under assignees in cases.

**Comment by weslambert** [comment, marked as answer]
An issue has been created to track this behavior: https://github.com/Security-Onion-Solutions/securityonion/issues/8184
